android/app/src/main/kotlin/com/jabook/app/jabook/compose/feature/player/PlayerScreen.kt

```
@@ -2040,24 +2253,54 @@ private fun PlayerContent(
+                                        release()
+                                    }
+                                    bookmarkPlayer.value = null
+                                    isPlayingBookmarkAudio = false
+                                }
+                                player.setOnPreparedListener {
+                                    it.start()
+                                    bookmarkPlayer.value = player
+                                    isPlayingBookmarkAudio = true
+                                }
+                                player.prepareAsync()
+                            } catch (e: java.io.IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)